apps/api/src/modules/job/job.service.ts

```diff
@@ -1,21 +1,43 @@
 import {
+  CreateJobDto,
   EditJobDto,
   GetJobCardWithMaxPageDto,
   JwtDto,
   SearchJobDto,
   UserType,
 } from '@modela/dtos'
 import { Injectable, NotFoundException } from '@nestjs/common'
-import { ForbiddenException } from '@nestjs/common/exceptions'
+import {
+  ForbiddenException,
+  InternalServerErrorException,
+} from '@nestjs/common/exceptions'
 
 import { JobRepository } from './job.repository'
 
 @Injectable()
 export class JobService {
   constructor(private repository: JobRepository) {}
 
-  create(createJobDto: EditJobDto) {
-    return 'This action adds a new job'
+  async createJob(createJobDto: CreateJobDto, userId: number) {
+    // if minAge is greater than maxAge, throw error
+    if (createJobDto.minAge > createJobDto.maxAge) {
+      throw new ForbiddenException('minAge is greater than maxAge')
+      return
```

Comment:
```suggestion
      throw new BadRequestException('minAge is greater than maxAge')
```
If throw don't have to return and this is bad request error not forbidden